### DISCUSSION TITLE
Будет ли работать Zapret в КНР или нет?

### DISCUSSION THREAD

**Discussion by ComBanan** [Flowseal/zapret-discord-youtube#897] [Q&A]
Думал по учебе поехать туда и возник вопрос, будет ли Запрет работать там? Если нет, то вы знаете схожие аналоги похожие на Zapret?

**Comment by testidcode** [comment]
По идее их "стена" блокирует не сами сайты, а все, кроме белого списка ip, поэтому не должен, иначе бы все кому там надо не под vpn сидели с пингом и повышенным трафиком (+ насколько я знаю с риском штрафов, ибо там vpn запрещен не только лишь на словах, за него, если поймают - ответственность), а через подобные более простые решения. 
Из подобных:
zapret by @bol-van (for MacOS, Linux and Windows)
Green Tunnel by @SadeghHayeri (for MacOS, Linux and Windows)
DPI Tunnel CLI by @zhenyolka (for Linux and routers)
DPI Tunnel for Android by @zhenyolka (for Android)
PowerTunnel by @krlvm (for Windows, MacOS and Linux)
PowerTunnel for Android by @krlvm (for Android)
SpoofDPI by @xvzc (for macOS and Linux)
SpoofDPI-Platform by @r3pr3ss10n (for Android, macOS, Windows)
GhosTCP by @macronut (for Windows)
ByeDPI for Linux/Windows + ByeDPIAndroid for Android (no root)
